scripts/build-rpm.sh

```diff
@@ -32,8 +32,19 @@ description='drush: Pantheon rpm containing commandline tool for Drupal'
 versions=$(cat $bin/../VERSIONS.txt | grep -v '^#')
 
 for version in $versions; do(
-	releasenum=${version%%.*}
-	releasever=${version%.*}
+	if [[ $version == "6-2-remove-uname" ]] || [[ $version == "5-10-production" ]]
```

Comment:
Use something like:
```version=$(echo $version | sed -e 's/\([^-]*\)-\([^-]*\)-.*/\1.\2/')```

That will give you 6.2 and 5.10.  If you want it to be semver-like, then we could add a zero:

```version=$(echo $version | sed -e 's/\([^-]*\)-\([^-]*\)-.*/\1.\2.0/')```

This should have no effect on versions that are already semver